### DISCUSSION TITLE
tab to accept / alt + . next button obstructing current line

### DISCUSSION THREAD

**Discussion by RoelStierum** [community/community#18254] [Copilot Conversations]
Hi all,

**Description**:
I am currently facing a weird issue. I am using visual studio enterprise v17.2.3 in combination with github copilot v1.27.0.0. This used to work fine but suddenly the suggestion navigation box appears on the line i'm typing on. In the attached screenshot I wrote a return statement and was trying to let copilot complete it.

**Steps to reproduce**:
Unknown, it did not happen after a update. Reinstall of copilot and visual studio did not work

**Expected behavior**:
To see the tab to accept / alt + . next button on the next line, not obstructing the current line where i'm typing at

![Screenshot 2022-06-10 112254](https://user-images.githubusercontent.com/60602417/173034932-750c2b84-5b40-486d-ba68-b50ec3166d3e.png)

other example: if statement blocked by the button
![Screenshot 2022-06-10 123551](https://user-images.githubusercontent.com/60602417/173047796-0e9279e2-ee94-4cd5-bc3d-5e28c7799e4c.png)

**Comment by MrKrishnaAgarwal** [comment]
try to press the Tab key above caps lock
or
or hover on it with the cursor and click on accept
![image](https://user-images.githubusercontent.com/100597998/173052368-b57326ac-1acb-4b86-bb4b-7320314cb5aa.png)
and
if nothing works try updating vs code to the latest version
I hope it helps

**Reply by RoelStierum** [reply]
@MrKrishnaAgarwal Thank you for your answer, but I think that you did not get my question.

Autocompletion does work, both via clicking and using tab but the button which says "tab to accept" and "alt + . next" obstructs my line. The suggestion appears under that button, so I cannot see it. It hovers over the suggestion.

As I have mentioned I already reinstalled my visual studio, and I am not using vs code but vs..

**Comment by MrKrishnaAgarwal** [comment]
Suggestion:
VS Code is the best editor and you will not face any issue in VS Code.
Install Link: https://code.visualstudio.com
I recommend you use this and you'll not face any issue.

Thanks and I hope this helps
Have a great day

**Reply by RoelStierum** [reply]
Best editor is a personal opinion. Fact is that I need to use vs, and it is certainly not bad. This is not an answer to my question/bug report, this is a workaround. I am personally more comfortable in vs code/jetbrains ide's but during my internship I need to use vs.. :S

**Reply by MrKrishnaAgarwal** [reply]
Sorry for that
It was my suggestion
